components/Modal/CoordinationSuccess/index.tsx

```diff
@@ -0,0 +1,75 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+import classNames from "classnames/bind";
+
+import { PAGE_PATH } from "@/constants/pagePath";
+
+import styles from "./CoordinationSuccess.module.scss";
+
+const cn = classNames.bind(styles);
+
+export default function CoordinationSuccessModal() {
+  const router = useRouter();
+
+  return (
+    //빈 div 하나는 gif 자리입니다.
+    <div className={cn("container")}>
+      <div className={cn("modal-header")}>
+        <div></div>
+        <div className={cn("modal-title")}>
+          <div className={cn("modal-title-main")}>투표 요청이 완료되었어요</div>
+          <div className={cn("modal-title-sub")}>
+            요청한 투표 현황은 <span className={cn("sub-span")}>마이페이지 - 일정 조율 현황</span>
+            에서 확인하실 수 있어요
+          </div>
+        </div>
+      </div>
+      <div className={cn("modal-body")}>
+        <div className={cn("modal-body-title")}>투표 마감 후 진행 안내</div>
+        <div className={cn("modal-body-main")}>
+          <div className={cn("modal-body-content")}>
+            <Image src="/images/heart-box.png" width={92} height={92} alt="하트 표시 상자" />
+            <div>
+              <div className={cn("modal-body-content-text")}>
+                하나의 일정이 <b>과반수 이상</b> 받은 경우
```

Comment:
```suggestion
                하나의 일정이 <b>과반수 이상</b> 받은 경우
```
p4: `<b>` 태그 좋네요👏👏👏
하지만 다른 태그쪽에서 `div`태그가 조금 많은 것 같아요 `p`, `span` 태그도 적절히 사용해주는 것도 좋을 것 같아요!